### DISCUSSION TITLE
Display null values other than zero

### DISCUSSION THREAD

**Discussion by SutharMukesh** [flant/grafana-statusmap#161] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17264175/56473887-4a3b1e00-648f-11e9-8a0b-6914b790109c.png)

I was trying docker input plugin of telegraf.
in which the status code gives 0 if the container is alive and gives null value for dead containers.
currently, the dead containers are displayed empty/invisible on the graph
is there any way to mark them as red colours?

**Comment by diafour** [comment, marked as answer]
The workaround is like this:
- Set «display nulls as zero» and read/dead for value 0
- Change a query to influx so 1 is returned instead of 0. Set green/alive for value 1.

**Comment by SutharMukesh** [comment]
how can I change the query for influxDb, as telegraf is responsible to store the metrics inside influx?

**Comment by diafour** [comment]
Please, show your query settings
